### PR TITLE
Scripts and tests

### PR DESCRIPTION
**Pull request by sergeyasad**

### CODE REVIEW

File: .github/workflows/merge.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+name: CI Merge
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  build:
+    # Setup matrix on Ubuntu, Windows and MacOS runners
```
Comment (vlad-nn) [inline .github/workflows/merge.yml:17]: Misleading comment: MacOS is not in matrix

File: mystreams.py [hunk 2]
```diff
@@ -320,7 +324,7 @@ def __init__(
         show_fps: bool = False,
         stream_depth: int = 10,
         allow_drop: bool = False,
-        multiplex = False,
+        multiplex=False,
```
Comment (vlad-nn) [inline mystreams.py:327]: Hmm, tonly this formatting gets changed. Did you format the file with Black as we do?

File: mystreams.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 from typing import Optional, Any, List, Union
 from contextlib import ExitStack
 
-import cv2
+import cv2, PIL
```
Comment (vlad-nn) [inline mystreams.py:18]: PIL is not used, why it is imported?

File: mytools.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
 #
 
 
-import sys, os, time, string, cv2, PIL
+import sys, os, time, string, cv2, PIL.Image
```
Comment (vlad-nn) [inline mytools.py:9]: In line 504(old)/516(new) PIL.Image is used. You change PIL import to PIL.Image import. Will it work?

File: test/conftest.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import pytest
```
Comment (vlad-nn) [inline test/conftest.py:1]: Need standard file header

File: test/test_notebooks.py [hunk 6]
```diff
@@ -0,0 +1,134 @@
+import nbformat
```
Comment (vlad-nn) [inline test/test_notebooks.py:1]: Need standard file header

File: test/test_notebooks.py [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+        + code_cells[0].source
+    )
+
+    nb = client.execute()
+    # save notebook with output, useful for debugging
+    # nbformat.write(nb, output_dir / f"{notebook_file.stem}.ipynb")
+    return nb
+
+
+_image_notebooks = [
```
Comment (vlad-nn) [inline test/test_notebooks.py:49]: Please put descrip[tive comment, what this table means and how to add new test

File: test/test_notebooks.py [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+    """Test notebook by executing it"""
+    filename = examples_dir / notebook_file
+    if input_file:
+        environ["CAMERA_ID"] = str(images_dir / input_file)
+
+    open_and_execute(filename)
+
+
+# currently skipped
+_skipped_notebooks = [
```
Comment (vlad-nn) [inline test/test_notebooks.py:128]: Put all test-definition-related stuff at the beginning of the file in one clearly marked section with step-by-step instructions how to add a test for new notebook

File: test/test_notebooks.py [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+        for image_count, image_datum in enumerate(image_data, 1):
+            cell_image = Image.open(BytesIO(base64.b64decode(image_datum)))
+            # cell_image.save(output_dir / f"{filename.stem}_{id}.{image_count}.png")
+            ref_image = Image.open(
+                reference_dir / f"{filename.stem}_{id}.{image_count}.png"
+            )
+            assert compare_ssim(cell_image, ref_image, GPU=False) > 0.975
+
+
+_imageless_notebooks = [
```
Comment (vlad-nn) [inline test/test_notebooks.py:110]: Put all test-definition-related stuff at the beginning of the file in one clearly marked section with step-by-step instructions how to add a test for new notebook

**Review by vlad-nn — CHANGES_REQUESTED** [review]

File: mystreams.py (hunk 2 above)
Comment (sergeyasad) [reply mystreams.py:327]: yes, it matches the formatting of the hintless 'show_ai_overlay=False' above clipped section

File: mytools.py (hunk 4 above)
Comment (sergeyasad) [reply mytools.py:9]: the plain PIL import didn't work for me, but PIL.Image does

File: .github/workflows/merge.yml (hunk 1 above)
Comment (sergeyasad) [reply .github/workflows/merge.yml]: removed macos from comment and ifs

File: mystreams.py (hunk 3 above)
Comment (sergeyasad) [reply mystreams.py]: removed import

File: mystreams.py (hunk 2 above)
Comment (vlad-nn) [reply mystreams.py:327]: lets add annotation then

**Review by vlad-nn — APPROVED** [review]

**Pull request merged**